jekyll/_cci2/runner-installation.adoc

```diff
@@ -365,6 +367,76 @@ sudo launchctl load '/Library/LaunchDaemons/com.circleci.runner.plist'
+
+The host needs to have Docker installed. Once the `runner` container is started, the container will immediately attempt to start running jobs. The container will be reused to run more jobs indefinitely until it is stopped.
+
+The number of containers running in parallel on the host is constrained by the host's available resources and your jobs' performance requirements.
+
+=== Create a Dockerfile that extends the CircleCI Runner image
+
+In this example, python3 is installed on top of the base image.
+
+`Dockerfile.runner.extended`
+
+```
+FROM circleci/runner:launch-agent
+RUN apt-get update; \
+    apt-get install --no-install-recommends -y \
+        python3
+```
+
+=== Build the Docker image
+
+```bash
+docker build --file ./Dockerfile.runner.extended .
+```
+
+=== Start the Docker container
+
+NOTE: The environment variable values are not to the `docker` command, so these environment variables are not visible in `ps` output.
+
+```bash
+CIRCLECI_RESOURCE_CLASS=<resource-class> CIRCLECI_API_TOKEN=<runner-token> docker run --env CIRCLECI_API_TOKEN --env CIRCLECI_RESOURCE_CLASS --name <container-name> <image-id-from-previous-step>
+```
+
+When the container starts, it will immediately attempt to start running jobs.
+
+=== Stopping the Docker container
+
+``` bash
+docker stop <container-name>
+```
+
+== Windows Installation
+
+NOTE: This has been tested for Windows Server 2019 and Windows Server 2016, both in Datacenter Edition with Desktop Experience.
+
+With this procedure, you install CircleCI Launch Agent and its dependencies, i.e., Chocolatey, Git and Gzip, on your Windows Server.
+
+Since this setup creates a new local administrator user that runs CircleCI jobs, your Windows Server needs to be capable of creating local users and accepting remote logons for them.
+
+* The Windows Server needs to be a domain member when you set this up in domain environments.The runner instance cannot run on a Windows Server operating as a domain controller.
+
+=== Installation Steps
+
+. Download the https://github.com/CircleCI-Public/runner-installation-files/tree/main/windows-install[`Install-CircleCIRunner.ps1` script] from Github to an easily accessible location. 
+
+. Open PowerShell as an administrator and navigate to the directory where you placed the script file.
+
+. Run the following in your PowerShell:
++
+```
+Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; 
+./Install-CircleCIRunner.ps1
+```
++
+The installation will be output into your PowerShell interface.
+
+. As part of the installation, the configuration file for the Runner (`launch-agent-config.yaml`) will open in Notepad. Please fill the file out with the requested information (see <<Configuration file reference, Configuration file reference>>). The configuration file is located in the installation directory - `C:\Program Files\CircleCI`, by default.
```

Comment:
```suggestion
. As part of the installation, the configuration file for the Runner (`launch-agent-config.yaml`) will opened in Notepad. Please fill the file out with the requested information (see <<Configuration file reference, Configuration file reference>>). The configuration file is located in the installation directory - `C:\Program Files\CircleCI`, by default.
```

Reply:
Hey @MaT1g3R, thanks for the review :) I am open to hear your thoughts on the change you suggested, but I think the way I have it is correct? Or what about `be opened`?

Reply:
whoops, I assumed there's a `be` there for some reason. You can ignore this.

Reply:
thank you! anything else standing in the way of a stamp?

Reply:
no, i think it's good!